app/experimenter/experiments/tests/test_forms.py

```diff
@@ -202,6 +205,302 @@ class Meta:
         self.assertEqual(change.old_status, old_status)
         self.assertEqual(change.new_status, new_status)
 
+    def test_mixin_sets_old_values_and_new_values(self):
+        experiment = Experiment()
```

Comment:
In this test you test the case of an empty experiment and an all fields form that sets all the fields and checks new values, and then a populated experiment with a single field form that checks the old and new values for that one field.  Could these be separate tests? What about the case of old_values for an all fields form on a fully populated experiment?

Reply:
will do